lambdas/service/app.py

```diff
@@ -85,6 +85,10 @@
         {
             'name': 'Health',
             'description': 'For checking on service health'
+        },
+        {
+            'name': 'DSS',
```

Comment:
We might want to be explicit about this endpoints being a `get` file proxy from the DSS aws replica.